app/services/katello/registration_manager.rb

```diff
@@ -294,14 +302,15 @@ def populate_subscription_facet(host, activation_keys, consumer_params, uuid)
         subscription_facet
       end
 
-      def remove_host_artifacts(host, clear_content_facet: true)
+      def remove_host_artifacts(host, clear_content_facet: true, kickstart_repository_id: nil)
```

Comment:
Can we get a logger debug statement here with the params, just in case support needs to debug a customer case, since we are clearing a lot of stuff from a host.

Reply:
Done :+1: